msteams-platform/tabs/how-to/dashboard-teams-toolkit.md

```diff
@@ -0,0 +1,991 @@
+---
+title: Create a dashboard in Teams
+author: v-silakshmi
+description:  Describes Dashboard, widget and Graph API call in Teams toolkit 
+ms.author: surbhigupta
+ms.localizationpriority: medium 
+ms.topic: Dashboard, widget, and Graph API call in Teams toolkit
+ms.date: 01/17/2023
+---
+
+# Create a dashboard in Teams
+
+Dashboards in the Teams toolkit allow you to monitor your business and view important metrics. A widget displays content from apps and services to stay engaged with the latest updates in Teams. Graph API can be used to visualize details about the implementation of the selected API.
+
+## Dashboard
+
+ A dashboard is a tool to track, analyze, and display data to gain insight of an organization or a specific process. The dashboard tab template from the Teams toolkit enables you to get started with integrating a canvas with multiple cards that provide an overview of the content in Microsoft Teams.
+
+Example of a dashboard:
+
+:::image type="content" source="../../assets/images/dashboard/dashboard-demonstration.png" alt-text="Screenshot shows the sample of a dashboard.":::
+
+**Advantages**
+
+* Dashboards connect numerous metrics, data sources, APIs, and services to help business extract relevant information from the sources and present it to the users.
+
+* Customizable dashboards allow your business to set specific goals that help you track the information you need to view in multiple areas and across departments.
+
+## Add a new dashboard
+
+Following are the steps to add a new dashboard layout:
+
+1. [Create a dashboard class](#create-a-dashboard-class)
+1. [Override methods to customize dashboard layout](#override-methods-to-customize-dashboard-layout)
+1. [Add a route for the new dashboard](#add-a-route-for-the-new-dashboard)
+1. [Modify manifest to add a new dashboard tab](#modify-manifest-to-add-a-new-dashboard-tab)
+
+### Create a dashboard class
+
+Create a file with the extension `.tsx` for your dashboard in the tabs/src/views/dashboards directory, for example, YourDashboard.tsx. Then, create a class that extends the Dashboard class:
+
+```typescript
+
+export default class YourDashboard extends Dashboard {}
+
+```
+
+export default class YourDashboard extends Dashboard {}
+
+### Override methods to customize dashboard layout
+
+Dashboard class provides few methods that you can override to customize the dashboard layout. The following table lists the methods that you can override:
+
+| **Methods** | **Function** |
+|---|---|
+| `rowHeight()` | Customize the height of each row of the dashboard. |
```

Comment:
This may be a typo, it should be `rowHeights()`

Reply:
Updated.